docs/docs/forms.md

```diff
@@ -114,8 +115,6 @@ class Form extends React.Component {
     return (
       <div>
         <input type="text"
-          placeholder="Hello!"
```

Comment:
Why do we remove the placeholder too? It is not related to controlled-ness of the components.